packages/server/.eslintignore

```diff
@@ -0,0 +1,5 @@
+database
+postgres/queries/generated
+segmentFns/*
+types/githubTypes.ts
+postgres/migrationTemplate.ts
```

Comment:
`.eslintignore` in root of parabol project dir was not working, so i broke it out into separate client and server files and put them in the root of their respective packages.